pykokkos/core/translators/bindings.py

```diff
@@ -156,26 +156,27 @@ def generate_functor_instance(functor: str, members: PyKokkosMembers) -> str:
     if len(args) == 0:
         args.append("0")
 
-    args.append(Keywords.RandPoolSeed.value)
-    args.append(Keywords.RandPoolNumStates.value)
+    if(with_random_args):
```

Comment:
You don't need the parentheses here, just `if with_random_args:`

Reply:
done in [c3f6f4](https://github.com/kokkos/pykokkos/pull/107/commits/c3f6f46b5e7f4b4e177211c99c7d0329e1708567)